R/render_debrief_doc.R

```diff
@@ -0,0 +1,26 @@
+#' Render debriefing doc
```

Comment:
Looks like this is very similar to the render_comms_doc. With a bit of refactoring, this could be a single function...